pkg/chartutil/files.go

```diff
@@ -207,11 +207,11 @@ func ToToml(v interface{}) string {
 	return b.String()
 }
 
-// ToJson takes an interface, marshals it to json, and returns a string. It will
+// ToJSON takes an interface, marshals it to json, and returns a string. It will
 // always return a string, even on marshal error (empty string).
 //
 // This is designed to be called from a template.
-func ToJson(v interface{}) string {
+func ToJSON(v interface{}) string {
```

Comment:
Technically, this is part of the public API, too... but its only actual usage is as a template function. So we might be able to get away with changing this one.